src/Microsoft.DotNet.Build.Tasks/GenerateTestExecutionScripts.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class GenerateTestExecutionScripts : Task
+    {
+        [Required]
+        public string[] TestCommands { get; set; }
+
+        [Required]
+        public string[] TestDependencies { get; set; }
+
+        [Required]
+        public string RunnerScriptWindowsTemplate { get; set; }
+
+        // Not used yet.
+        [Required]
```

Comment:
Do we even need 2 required parameters for the templates? We could just have the one template property and the target can pass in the appropriate one.